packages/palette/src/ui-colors.less

```diff
@@ -1,35 +1,35 @@
-@leafygreen__green--dark-3: #0B3B35;
+@leafygreen__green--dark-3: #0b3b35;
 @leafygreen__green--dark-2: #116149;
-@leafygreen__green--base: #13AA52;
-@leafygreen__green--light-2: #C3E7CA;
-@leafygreen__green--light-3: #E4F4E4;
+@leafygreen__green--base: #13aa52;
+@leafygreen__green--light-2: #c3e7ca;
+@leafygreen__green--light-3: #e4f4e4;
 
-@leafygreen__gray--dark-3: #21313C;
-@leafygreen__gray--dark-2: #3D4F58;
-@leafygreen__gray--dark-1: #5D6C74;
-@leafygreen__gray--base: #89989B;
-@leafygreen__gray--light-1: #B8C4C2;
-@leafygreen__gray--light-2: #E7EEEC;
-@leafygreen__gray--light-3: #F9FBFA;
+@leafygreen__gray--dark-3: #21313c;
+@leafygreen__gray--dark-2: #3d4f58;
+@leafygreen__gray--dark-1: #5d6c74;
+@leafygreen__gray--base: #89979b;
+@leafygreen__gray--light-1: #b8c4c2;
+@leafygreen__gray--light-2: #e7eeec;
+@leafygreen__gray--light-3: #f9fbfa;
 
-@leafygreen__blue--dark-3: #0D324F; 
-@leafygreen__blue--dark-2: #1A567E; 
-@leafygreen__blue--base: #007CAD; 
-@leafygreen__blue--light-1: #9DD0E7; 
-@leafygreen__blue--light-2: #C5E4F2; 
-@leafygreen__blue--light-3: #E1F2F6; 
+@leafygreen__blue--dark-3: #0d324f;
+@leafygreen__blue--dark-2: #1a567e;
+@leafygreen__blue--base: #007cad;
+@leafygreen__blue--light-1: #9dd0e7;
+@leafygreen__blue--light-2: #c5e4f2;
+@leafygreen__blue--light-3: #e1f2f6;
 
-@leafygreen__yellow--dark-3: #543E07; 
-@leafygreen__yellow--dark-2: #86681D; 
-@leafygreen__yellow--base: #FFDD49; 
-@leafygreen__yellow--light-2: #FEF2C8; 
-@leafygreen__yellow--light-3: #FEF7E3; 
+@leafygreen__yellow--dark-3: #543e07;
+@leafygreen__yellow--dark-2: #86681d;
+@leafygreen__yellow--base: #ffdd49;
+@leafygreen__yellow--light-2: #fef2c8;
+@leafygreen__yellow--light-3: #fef7e3;
 
-@leafygreen__red--dark-3: #570B08; 
-@leafygreen__red--dark-2: #8F221B; 
-@leafygreen__red--base: #CF4A22; 
-@leafygreen__red--light-2: #F9D3C5; 
-@leafygreen__red--light-3: #FCEBE2; 
+@leafygreen__red--dark-3: #570b08;
```

Comment:
Why were these all lowercased?

Reply:
happened when i ran my formatter

Reply:
Wow that's awful, and I don't know why it hasn't formatted this file in the past...

We use all uppercase in JS. Is there a rule we can change?

Reply:
Looking into it

Reply:
Did it directly in Githubs UI since my editor was not having it